### ISSUE TITLE
Java. Bad color display

### ISSUE THREAD

**Issue by KernelMrex** [atom/atom#16796]
This code displays with errors.

```java
try(FileInputStream fin1 = new FileInputStream(args[0]); FileInputStream fin2 = new FileInputStream(args[1])) {
    int b1, b2;
    do {
        b1 = fin1.read();
        b2 = fin2.read();
    } while(b1 == b2 && (b1 != -1 || b2 != -1));
    if (b1 != b2) {
        System.out.println("Different files!");
        return;
    }
    System.out.println("Files has no differences");
}
catch (FileNotFoundException exc) {
    System.out.println("Error: File not found!");
    System.out.println("Use \"java CompFiles <File1> <File2>\"");
}
catch (IOException exc) {
    System.out.println(exc);
}
```
![image](https://user-images.githubusercontent.com/24671528/36435936-06ae9974-1674-11e8-9d47-eaccf584fe11.png)

**Comment by Arcanemagus** [comment]
Duplicate of https://github.com/atom/language-java/issues/88.

It looks like there is a PR already up fixing this, just waiting on a review from @50Wliu.

**Comment by Ben3eeE** [comment]
Thanks @Arcanemagus 

Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/atom/language-java/issues/88. You may want to subscribe there for updates.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
